blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/HttpAggregatedMultiIngestionHandler.java

```diff
@@ -62,14 +65,32 @@ public void handle(ChannelHandlerContext ctx, FullHttpRequest request) {
 
         Tracker.getInstance().track(request);
 
+        requestCount.inc();
+
+        if ( !mediaTypeChecker.isContentTypeValid(request.headers()) ) {
+            DefaultHandler.sendResponse(ctx, request,
+                    String.format("Unsupported media type for Content-Type: %s", request.headers().get(HttpHeaders.Names.CONTENT_TYPE)),
+                    HttpResponseStatus.UNSUPPORTED_MEDIA_TYPE
+            );
+            return;
+        }
```

Comment:
It would be nice if we didn't have to replicate these if statements.  It's a little weird with the return statement, even if we encapsulated it within a method, we'd still have to check if the calling code should return or not.  

My vote is to pull into a static method or parent class.  The other devs can chime in.